libs/core/langchain_core/chat_history.py

```diff
@@ -45,6 +51,10 @@ def clear(self):
     def add_user_message(self, message: Union[HumanMessage, str]) -> None:
         """Convenience method for adding a human message string to the store.
 
```

Comment:
@baskaryan is it safe to use the deprecation functionality here. I want to mark this as pending deprecation rather than deprecated. Wasn't sure where we are right now in terms of surfacing warnings -- I don't think this should be surfaced to users especially if it's not directly used by user code

Reply:
but also is it so bad to keep these around as convenience methods? think more important thing would just be to emphasize/only show add_messages in docs